### DISCUSSION TITLE
Switch to window.

### DISCUSSION THREAD

**Discussion by squid46** [php-webdriver/php-webdriver#1071] [General]
Hello.
How do I switch to the web browser window? I start webdriver on command via websocket. Firefox starts, but it is not active in the tray. I keep the browser from which I sent the request active.

**Comment by ernstki** [comment]
It would probably help if you were to share some information about your platform (Windows, I guess?) and some sample code, if you can.

For example, I understand what the words "webdriver" and "websocket" _mean_, but it's hard to visualize how you did that without seeing code.
